x-pack/plugins/reporting/server/browsers/chromium/driver/screenshot_stitcher/combine.js

```diff
@@ -42,6 +43,9 @@ export function $combine(screenshots, outputDimensions) {
   const output$ = pngs$
     .reduce(
       (output, { dimensions, png }) => {
+        logger.debug(`Output dimensions is ${JSON.stringify(outputDimensions)}`);
+        logger.debug(`Input png w: ${png.width} and h: ${png.height}`);
+        logger.debug(`Creating output png with ${JSON.stringify(dimensions)}`);
```

Comment:
@kobelb - and maybe here - this next function call sometimes throws an error about reading outside the bounds of the image.

Reply:
lgtm